src/Microsoft.ML.Data/Transforms/ColumnCopying.cs

```diff
@@ -233,15 +233,21 @@ protected override DataViewSchema.DetachedColumn[] GetOutputColumnsCore()
 
             public void SaveAsOnnx(OnnxContext ctx)
             {
-                var opType = "CSharp";
+                var opType = "Identity";
 
                 foreach (var column in _columns)
                 {
-                    var srcVariableName = ctx.GetVariableName(column.inputColumnName);
-                    _schema.TryGetColumnIndex(column.inputColumnName, out int colIndex);
+                    var inputColumnName = column.inputColumnName;
+                    if (!ctx.ContainsColumn(inputColumnName))
+                    {
+                        ctx.RemoveColumn(inputColumnName, false);
+                        continue;
+                    }
+
```

Comment:
Was this branch tested? It seems not because this code seems buggy to me as RemoveColumn tries to remove inputColumnName from  columnNameMap but ctx.ContainsColumn(inputColumnName) tells us this key is not present if it is false. It would also be nice if you synced latest to master so that we can see code coverage on this. 

Remove Column is defined as:
        public override void RemoveColumn(string colName, bool removeVariable)
        {
            _host.CheckNonEmpty(colName, nameof(colName));

            if (removeVariable)
            {
                foreach (var val in _intermediateValues)
                {
                    if (val.Name == _columnNameMap[colName])
                    {
                        _intermediateValues.Remove(val);
                        break;
                    }
                }
            }
            _columnNameMap.Remove(colName);
        }

and ContainsColumn(string colName) => _columnNameMap.ContainsKey(colName);